cmd/controller-manager/utils.go

```diff
@@ -11,16 +11,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
```

Comment:
I wonder why we don't use `controller / reconciler` pattern in `watchConfig`. Inside the `controller`, it's a `RateLimitQueue` :(

I will find the problem or replace it into a controller.

Reply:
> why we don't use `controller / reconciler`.

Because https://github.com/kubernetes-sigs/controller-runtime/issues/244 hasn't landed, we cannot listen to `ConfigMap` with a specified label. Listening to all `ConfigMap` on the cluster is a disaster for cache and network.

I can try to use `RateLimitQueue` to replace `Coalescer` here.